### PR TITLE
added some quick installation instructions

### PR DESCRIPTION
**Pull request by Lyla-Fischer**
When installing the XBlock SDK, I thought that it would help future new contributors if I clarified some of the advice given in the readme.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Lyla-Fischer! I've created [OSPR-862](https://openedx.atlassian.net/browse/OSPR-862) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).
We can't start reviewing your pull request until you've   added yourself to the [AUTHORS](https://github.com/edx/xblock-sdk/blob/master/AUTHORS) file . Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

File: README.rst [hunk 1]
```diff
@@ -16,6 +16,8 @@ This code runs on Python 2.7.
 1.  Get a local copy of this repo.
 
 2.  (Optional)  Create and activate a virtualenv to work in.
+        
+        A virtualenv allows you to isolate your development dependencies for this project from any other dependencies that you might have for other projects. If you do any tinkering at all, it will save you a lot of headaches to install a virtualenv to work in. Read more about it here: http://docs.python-guide.org/en/latest/dev/virtualenvs/
```
Comment (nedbat) [inline README.rst]: Nice. But please hard-wrap the lines to match the rest of the file.

File: README.rst [hunk 2]
```diff
@@ -94,7 +96,8 @@ written in a simple XML language.  Note this is not an XML format we are
 proposing as a standard.
 
 Once you install your XBlock into your virtualenv, the workbench will
-automatically display its scenarios for you to experiment with.
+automatically display its scenarios for you to experiment with. You don't have to use anything more complicated than pip to install the package:
```
Comment (nedbat) [inline README.rst]: If you end this line with two colons, it will make a nice display of the following line.

File: README.rst (hunk 1 above)
Comment (nedbat) [inline README.rst]: Also, I'm not sure of the right indentation to use for this paragraph to be included in bullet 2.  I check these things by pasting the .rst text into http://rst.ninjs.org/

Comment (Lyla-Fischer) [inline README.rst]: Thanks for the link. That will be useful in the future in addition to this contribution.

Comment (nedbat) [inline README.rst]: Looks like this does need to be out-dented a bit to match the numbered paragraph.

File: README.rst (hunk 2 above)
Comment (nedbat) [inline README.rst]: Also, looks like I was wrong here: use one colon and add a blank line, and the rst editor will be happier.

**Comment by nedbat** [comment]
It would have been better to finish this change and merge it here.  The other pull request has two distinct issues, with very different levels of controversy.

**Pull request closed without merging**